tls/s2n_psk.h

```diff
@@ -50,6 +68,9 @@ struct s2n_psk_parameters {
     s2n_psk_key_exchange_mode psk_ke_mode;
 };
 
+/* This function will become an extern function and moved to s2n.h once we release the psk API. */
```

Comment:
"extern" means nothing in relation to functions. It'll become a publicly visible API; you could specify we'll label it with "S2N_API".